ui/src/search.ts

```diff
@@ -0,0 +1,35 @@
+import { writable } from "svelte/store";
+
+import * as event from "./event";
+
+import { ValidationStatus } from "./validation";
+
+interface UntrackedProject {
+  urn: string;
+}
+
+enum Kind {
+  Update = "UPDATE",
+}
+
+interface Update extends event.Event<Kind> {
+  kind: Kind.Update;
+  urn: string;
+}
+
+type Msg = Update;
+
+const update = (msg: Msg): void => {
+  switch (msg.kind) {
+    case Kind.Update:
+      validation.set({ status: ValidationStatus.Loading });
+      setTimeout(() => {
+        validation.set({ status: ValidationStatus.Success });
+      }, 1000);
```

Comment:
Is this to fake the validation?

Reply:
Genau - this PR is supposed to be the visuals only so @brandonhaslegs can make sure the flow looks as it should. I want to do the proxy side in another PR to keep things short & sweet - I could add a very basic validation here to ensure the input isn't empty, but because we don't have networking yet, I can't validate that the `urn` actually does anything 😄 .